Tutorial HTML/Html 01/README.md

```diff
@@ -0,0 +1,85 @@
+
+HTML este un limbaj de interpretare.
+Acest lucru ne spune ca HTML e un format de text cu anumite componente ce sunt interpretate de browser pentru a fi afisate in mod corespunzator.
+
+De exemplu ne putem gandi aici la relatia dintre un cititor si un poet.
+Poetul scrie o poezie de sucess cu gandul la dragoste, si fericire, dintr-o experienta trecuta.
+Vine citirorul, se apuca de citit si datorita experientelor sale diferite acesta interpreteaza poezia ca fiind una trista, si o povesteste mai departe ca atare.
+In cazul de fata cititorul nostru este browserul, iar scriitorul fiind cel care creaza o pagina web
+
+Un fisier HTML contine comunica cu browserul prin taguri *(mai corect spus browserul ia aceste taguri si le interpreteaza ca atare)* 
+
+### Taguri HTML
+Un element HTML este format din un `tag de inceput` niste text si un `tag de final`
+Ele au rolul de a specifica browserului modul in care dorim ca textul pe care il dam sa fie interpretat
+De exeplu `<p> Sunt un paragraf ! </p>` va face browserul sa interpreteze acest text ca un paragraf
+
+- Un tag HTML este un text de forma `<` `final?` `numetag` `optiuni` `>`
+- Caracterele `<>` definesc inceputul si finalul unui tag
+- `final?` este defapt caracterul `/` care pus inaintea denumirii unui tag specifica faptul ca tagul dat este un tag de final
+- `numetag` poate fi orice text fara spatii folosind doar caracterele de la `a-z` `A-Z` `0-9` 
+- **Pentru a inchide un tag `numetag` definit la tagul de inceput trebuie sa fie acelasi ca `numetag` de la tagul de final**
+(`-` *specifica toate caracterele dintre din interval. Ex `a-z` specifica toate caracterele  litere mici de la `a` la `z`*)
+- `optiuni` un set de proprietati ce pot fi asignate unui tag
+
+Cu alte cuvinte un tag HTML este orice text de forma `<numetag>`, iar un element HTML este orice text de forma `<numetag> un text random </numetag>`
+
+#### Sa trecem la practica
+
+Pentru inceput putem crea un folder pe desktop in care vom avea pagina noastra web.
+In interiorul folderului cream un nou fisier, cu ce denumire dorim, dar cu extensia `.html` (ex `test.html`)
+
+Putem observa cum calculatorul ne indica deja faptul ca fisierul creat poate fi deschis in browser
+Daca deschidem acest fisier vom vedea o pagina goala, pentru ca nu am adaugat nimic inca.
+
+Vom deschide acest fisier in notepad 
+##### *( `click dreapta` > `open with`  > `notepad`. Daca intampinati probleme cu acest lucru puteti sari la sectiunea [editor de cod]() unde va voi ajuta cu acest lucru )*
+
+Vom oferi browserului niste text pe care il poate interpreta
+Un fisier HTML incepe cu tagul `<html>` si se termina cu `</html>` care delimiteaza inceputul si sfarsitul unui fisier html.
+Un fisier HTML mai contine inca un tag in interiorul acestuia si anume `<body></body>` in care vom pune alte taguri, cu text.
+In interiorul acestui tag vom pune de exemplu un paragraf (`<p>paragraf</p>`) cu textul `Hello world !`
+
+     <html>
+    	<body>
```

Comment:
What about `head`? Si hai sa mai facem inca 2 sub-capitole, cu rolul/ce punem in `<head>` si `<body>`.